src/js/components/Baani.js

```diff
@@ -116,6 +116,26 @@ export default class Baani extends React.PureComponent {
     }
   };
 
+  makeSelection = e => {
+    const shareDivs = document.querySelectorAll('.share');
+    shareDivs.forEach(function(sd) {
+      sd.style.display = 'none';
+    });
```

Comment:
Use arrow function

Reply:
Removed the function. Using a separate class to show.